src/components/transactions/Swap/SwapModalContent.tsx

```diff
@@ -224,6 +226,12 @@ export const SwapModalContent = ({
         </Typography>
       )}
 
+      {swapTarget.reserve.symbol === 'stETH' && (
```

Comment:
The failure is on the withdraw so this should be `swapSource`, right @grothem ?

Reply:
This is a warning. `Swap` button is disabled in `stETH` collateral position with the changes of this PR. Since you can swap collateral from other asset to `stETH` this is warning you that if you do it you'll be limited to certain actions with your new `stETH` collateral